### PR TITLE
Fixed .travis.yml (fixes #24)

### LINKED ISSUE #24: Travis build fails because it can't install SciPy
https://github.com/travis-ci/travis-ci/issues/2890

https://travis-ci.org/marco-c/crashsimilarity/builds/212056954

### PR DESCRIPTION
**Pull request by ray-pankaj**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,3 +1,7 @@
 language: python
-install: "pip install -r requirements.txt"
+install:
+    - pip install --upgrade pip setuptools wheel
+    - pip install --only-binary=numpy,scipy numpy
```
Comment (marco-c) [inline .travis.yml]: Nit: `numpy` is listed twice 😄

Comment (ray-pankaj) [inline .travis.yml]: pip install has to have one requirement to install

Comment (marco-c) [inline .travis.yml]: Doesn't `pip install --only-binary=numpy,scipy` work?

Comment (ray-pankaj) [inline .travis.yml]: Yup, it does. Thanks! :)

**Pull request merged**